### DISCUSSION TITLE
So how do I disable it?

### DISCUSSION THREAD

**Discussion by Shineblind** [emoose/DLSSTweaks#125] [Q&A] [answered]
what Ive tried,

removing all dlsstweaks related files from the game directory, 
disable Force DLAA in dlsstweaks,
enable DLSSQualityLevels
run  DisableNvidiaSigOverride,

I can ofc disable it in game (I installed it for Ready or Not), by turning the DLSS slider to off, however, the Anti Aliasing slider gets pretty much disabled by DLSSTweaks it seems.

edit: I would also like to report it works in the game Ready or Not, and only gives the laser attachment in game a very slight ghosting

edit2: tried reinstalling the game, I still cant remove it, AA slider is disabled even though it has a value in the game INI file, 
tried wiping the win64 folder and verify files, its still not gone somehow

**Comment by Ezeky3l** [comment]
Did you edit "NvGlobalProfileSettings" with DLSSTweaksConfig.exe ? I quote emoose : "GlobalForceDLAA: if set to true, all DLSS quality levels will be forced as DLAA instead, on all DLSS 3.1.11+ games (without DLSSTweaks needing to be setup on them)".
If you don't I can't help, I never had this issue in any game with DLSSTweaks...

**Reply by Shineblind** [reply]
pretty sure I didnt touch those, heres my settings https://i.imgur.com/vOaH3rL.png

**Reply by Ezeky3l** [reply]
Your settings are correct. I don't know what's going on, that's weird. Try a clean install of your game.

**Comment by Shineblind** [comment, marked as answer]
oh okay, I had some weird line in the engine.ini, I might have put it there, I keep up this thread anyway just in case someone ends up doing the same.

the line turned off AA
